cmd/Makefile

```diff
@@ -191,7 +191,11 @@ endif # !CONFIG_SPL_BUILD
 obj-$(CONFIG_$(SPL_)CMD_TLV_EEPROM) += tlv_eeprom.o
 
 # core command
+ifndef CONFIG_SPL_BUILD
```

Comment:
this change looks pretty suspucious (however it fixed linking issues).
Maybe I've broken something that caused nvedit compilation for SPL even with disabled ENV support (`CONFIG_SPL_ENV_SUPPORT`)

Reply:
We shouldn't need this as a compile fix.  Please re-test w/ the config fragments we've shared.